### ISSUE TITLE
Some fixes

### ISSUE THREAD

**Issue by KrzysztofGoralski** [knoldus/akka-http-slick#1]
Hi, I forked this repo and did some fixes. You may check it here:
https://github.com/KrzysztofGoralski/akka-http-slick-guice

- Guice added
- Changed Routes to Get, Put, Post, Delete and urls
- Changed Json Parser from Lift to akka-http/Spray one

**Comment by satendrakumar06** [comment]
@KrzysztofGoralski Great 111

**Comment by satendrakumar06** [comment]
@KrzysztofGoralski if you want update this repo by your code, Please create pull request, I will merge it.

**Comment by KrzysztofGoralski** [comment]
@satendrakumar06  Ok. We can do that :) 

Forgot that I could do that and contributed my fork to lightbend. I was too fast about it ;) Sorry. https://www.lightbend.com/activator/template/akka-http-slick-guice

So, I will ask to delete it from lightbend and switch to your template only (?)

**Comment by satendrakumar06** [comment]
@KrzysztofGoralski  If you already  contributed to lightbend Then I would to suggest don't delete it from lighbend. So we don't need to merge it.

If you still want to update it please squash several git commits into a one or two commit.
I really appreciated you work. Thanks

**Comment by KrzysztofGoralski** [comment]
Hi @satendrakumar06 If it is ok for you we can leave it like it is. Easier for me :) 

But If you want, we can merge. I feel that I should ask you before contributing it to lightbend because I just forked it and add some features and nothing more (I really liked your project too). Thats why I created this 'issue' to tell you about it.

I'm quite new to github and didn't really know that they will put it on lightbend page so fast.  I thought it will take some time ;)
